src/ode/ode/src/ode_MT/util_MT.cpp

```diff
@@ -253,4 +254,5 @@ void util_MT::checkClusterNodes(int clusterCount, dxClusterNode** clusterAABBs)
             prevNode = node;
         }
     }
+#endif
 }
```

Comment:
```suggestion
}
#endif
```
And we should also add some #ifndef dNODEBUG in the `util_MT.h` file around this function declaration.